recipes/dismod-at/build.bat

```diff
@@ -0,0 +1,52 @@
+:: echo
+echo off
+
+:: build
+mkdir build
+cd build
+
+:: PKG_CONFIG_PATH
+set PKG_CONFIG_PATH=^
+%BUILD_PREIX%\Library\lib\pkgconfig;^
+%BUILD_PREIX%\Library\share\pkgconfig;
```

Comment:
This is not correct. pkg-config should not point to BUILD_PREFIX. Please remove this.

Reply:
@xhochy 
> Please remove all the comments here. They seem to be quite "empty" .
> This is not correct. pkg-config should not point to BUILD_PREFIX. Please remove this.

Done in the following commit:
https://github.com/conda-forge/staged-recipes/pull/29516/commits/b44d02ff